packages/wallet/backend/src/account/controller.ts

```diff
@@ -53,13 +52,30 @@ export class AccountController implements IAccountController {
     const userId = req.session.user.id
 
     try {
-      const getAccountsResult = await this.deps.accountService.getAccounts(
-        userId
-      )
+      const accounts = await this.deps.accountService.getAccounts(userId)
 
       res
         .status(200)
-        .json({ success: true, message: 'SUCCESS', data: getAccountsResult })
+        .json({ success: true, message: 'SUCCESS', data: accounts })
+    } catch (e) {
+      next(e)
+    }
+  }
+
+  listAccountsWithGraphsFetched = async (
+    req: Request,
+    res: CustomResponse<Account[]>,
+    next: NextFunction
+  ) => {
+    const userId = req.session.user.id
+
+    try {
+      const accounts =
+        await this.deps.accountService.getAccountsWithGraphFetched(userId)
+
+      res
+        .status(200)
+        .json({ success: true, message: 'SUCCESS', data: accounts })
```

Comment:
Instead of having a separate function for this, should we have a query param (`include`) where we provide what else need to be fetched? In our case it will be something like `http://localhost/accounts?include=paymentPointers`.